assets/js/components/notifications/data-error.js

```diff
@@ -41,14 +42,15 @@ import InvalidCredentialsWarning from './invalid-credentials-warning';
  * @param {boolean} inGrid     Creates layout to fit within an existing grid with 'cell' classes. Default is half-width grid cells. Default: false.
  * @param {boolean} fullWidth  Creates layout with 'cell--span-12' to be full width. Default: false.
  * @param {boolean} createGrid Adds a full grid layout with padding. Default: false.
+ * @param {Object}  errorData  Error related data.
```

Comment:
This might be more accurately named as `errorObj` or `wpError` as `errorData.data` is a bit confusing.

Alternatively we could rename `error` to `message` or `errorMsg` and `errorData` could be `error`.

Reply:
Updated. Used `errorObj` name.